docs/tutorial.md

```diff
@@ -481,32 +557,38 @@ conflicts.
 
 Let's try one final command for changing the contents of an exiting commit. That
 command is `jj diffedit`, which lets you edit the contents of a commit without
-checking it out.
+checking it out (i.e. the "right side").
```

Comment:
Since the default diff editor doesn't let you edit the contents of the right side, I think it's better to not add that. Also, the existing "lets you edit the contents of a commit" is probably better as "lets you edit the changes in a commit".

Reply:
Done in https://github.com/martinvonz/jj/commit/b2692a325f251969f212153df4d02b26b772efb0